packages/gui/src/components/app/AppKeyringMigrator.tsx

```diff
@@ -114,12 +121,26 @@ export default function AppKeyringMigrator() {
     );
   }
 
+  function handleKeyDown(e: KeyboardEvent): void {
+    if (e.getModifierState("CapsLock")) {
+      setShowCapsLock(true);
+    }
+  }
+
+  const handleKeyUp = (event) => {
```

Comment:
Suggestion for consistency with the `handleKeyDown()` function

```suggestion
  function handleKeyUp(e: KeyboardEvent): void {
```